guardian/media-atom-maker
# Fix YouTube furniture keyword bug

## What does this change?
Upgrading to v4 in a previous PR seemed to break the YT furniture Keyword function, as it would break trying to render the object `value` as a Component rather than a string. 

This PR conditionally either returns the ID or webTitle of the value, which appear to be the same when testing locally but may differ.

## How to test

Go to YouTube Furniture, click Edit, add a Keyword (and press enter). You should see the keyword pop up in the box as normal.

## How can we measure success?

YouTube Furniture Keywords can be added.
